### DISCUSSION TITLE
Plans to support Theia in Eclipse Che

### DISCUSSION THREAD

**Discussion by Wosin** [eclipse-theia/theia#13789] [General]
Hey, 
I've got a question regarding support of Theia in Eclipse Che. For a long time, Theia was the default editor used in Che, however, in the recent versions the support has been removed and the Che team doesn't plan to support it anymore due to limited resources. I think that having Theia running on Che is still valid use-case for many teams (including mine), so I wanted to ask if the Theia team plans to support resources (devfile, docker images, docs) allowing to run Theia on Che.

**Comment by msujew** [comment]
Hey @Wosin,

Theia project lead here. There's currently no interest from our end to support Che. I obviously can't speak for everyone, but as far as I can tell, most adopters prefer slimmer solutions to workspace management. AFAIK Che is quite the investment (from an infrastructure perspective). There are alternatives, a lot of adopters use stuff like [theia-cloud](https://theia-cloud.io/), whereas others roll their own workspace solution.

Maybe @JonasHelming can also chime in on that as well.

**Comment by JonasHelming** [comment]
Hi @Wosin ,

thank you for your message. I would find it interesting if Che would continue to support Theia, but this is really up to the Che team. Theia provides the Theia IDE, a standard IDE based on Theia, which can be used for this use case. It is also available as a Docker image (https://github.com/eclipse-theia/theia-blueprint?tab=readme-ov-file#docker). As the Theia IDE is actively maintained by the Theia community, it removes a lot of work from anyone who wants to host/provide a standard IDE based on Theia, e.g. in Che.
I belive the Theia IDE has not been available when Che decided to drop Theia, so maybe it is worth that you file this request in the Che project again.

Best regards,
Jonas

**Comment by virtualirfan** [comment]
Out of interest, what is the base technology layer for the online trial version [here](https://try.theia-cloud.io/) on theia-cloud.io?

**Reply by JonasHelming** [reply]
That is Theia Cloud: https://theia-cloud.io/
